### PR TITLE
Cccp 4552 - Update installation process

### PR DESCRIPTION
**Pull request by ajdelgado**
Update installation process to current versions of installers and packages

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -32,26 +32,26 @@
     mode: '0755'
   when: dell_dsu_rollout_gpg_keys
 
-- name: copy gpg keys
-  copy:
-    dest: /usr/libexec/dell_dup/{{ item }}
-    src: "{{ item }}"
-  loop: "{{ dell_dsu_gpg_keys }}"
-  when: dell_dsu_rollout_gpg_keys
+# - name: copy gpg keys
```
Comment (khappone) [inline tasks/main.yml]: These could be removed instead of commented.

File: tasks/main.yml [hunk 2]
```diff
@@ -11,8 +11,8 @@
   when: dell_dsu_manage_multipath_config
 
 - name: Install Dell DSU repo
-  environment: "{{ proxy_env|default({}) }}"
-  shell: "curl -O https://linux.dell.com/repo/hardware/dsu/bootstrap.cgi && yes {{ 'y' if dell_dsu_rollout_gpg_keys_from_dell_script else 'n' }} | bash bootstrap.cgi"
+  environment: "{{ proxy_env | combine( { 'IMPORT_GPG_CONFIRMATION': 'yes' } ) | default({}) }}"
```
Comment (khappone) [inline tasks/main.yml]: This works, but I don't know why as I feel the script should overwrite it.

**Review by khappone — CHANGES_REQUESTED** [review]

File: tasks/main.yml (hunk 1 above)
Comment (ajdelgado) [reply tasks/main.yml]: fixed

File: tasks/main.yml (hunk 2 above)
Comment (ajdelgado) [reply tasks/main.yml]: Changed to replace the specific variable default to yes (so it will import GPG keys)

**Review by khappone — APPROVED** [review]

**Pull request merged**